tests/blas_like/ColumnNorms.cpp

```diff
@@ -11,28 +11,30 @@
 using namespace El;
 
 template <typename T, DistWrap W>
-void TestColumnTwoNorms(Int m, Int n, const Grid& g, bool print) {
+void TestColumnTwoNorms(Int m, Int n, const Grid& g, bool print)
+{
   // Generate random matrix to test.
   DistMatrix<T, MC, MR, W> A(g);
   Uniform(A, m, n);
-  if (print) {
+  if (print)
     Print(A, "A");
-  }
   DistMatrix<T, MR, STAR, W> norms(g);
   ColumnTwoNorms(A, norms);
-  if (print) {
+  if (print)
     Print(norms, "norms");
-  }
-  for (Int j = 0; j < A.LocalWidth(); ++j) {
+  for (Int j = 0; j < A.LocalWidth(); ++j)
+  {
     T got = norms.GetLocal(j, 0);
     T expected = 0;
-    for (Int i = 0; i < A.LocalHeight(); ++i) {
+    for (Int i = 0; i < A.LocalHeight(); ++i)
+    {
       T val = A.GetLocal(i, j);
       expected += val * val;
     }
     expected = mpi::AllReduce(expected, g.ColComm());
     expected = Sqrt(expected);
-    if (Abs(got - expected) > 1e-5) {
+    if (Abs(got - expected) > 10 * limits::Epsilon<El::Base<T>>())
```

Comment:
Just want to clarify: The new check should be `Abs(got - expected) / std::max(expected, 1) > m * n * 10 * limits::Epsilon<El::Base<T>>()`?